crates/lib/src/qpu/api.rs

```diff
@@ -466,8 +479,9 @@ impl ExecutionOptions {
         }
     }
 
+    /// Get the [`cancel_controller_jobs_request::Target`] for the given quantum processor ID.
```

Comment:
Side question: it wouldn't be possible to force `prost` to only generate one `Target` type, would it?

Reply:
I think so? Each request type embeds its own `Target`, which results in a different model being exported for each. I'd assume you could lift that specification up and re-use across each request type.